app/models/operatingsystems/solaris.rb

```diff
@@ -6,7 +6,7 @@ def file_prefix
   end
 
   # sets the prefix for the tftp files based on the OS
-  def pxe_prefix(architecture = nil)
+  def pxe_prefix(architecture = nil, host = nil)
```

Comment:
This does not match the superclass signature, is this on purpose?

Reply:
Matched the superclass.